lib/training/wiki_slide_parser.rb

```diff
@@ -50,6 +51,12 @@ def remove_noinclude
     @wikitext.gsub!(%r{<noinclude>.*?</noinclude>\n*}m, '')
   end
 
+  # Category tags are useful for categorizing pages, but we don't want them to show up in the slides
+  # Example: [[Category:Programs & Events Dashboard]]
+  def remove_category
+    @wikitext.gsub!(%r{\[\[Category:.*?\]\]\n*}m, '')
+  end
```

Comment:
This should have a test.

Alternatively, we could just wrap the categories in noinclude tags and remove this method, which maybe makes more sense anyway in this context. (I can imagine trainings where we want to show how to add categories, and so unparsed category markup might appear on the slide, which would then get incorrectly removed by this code.)